dist/contract.js

```diff
@@ -31,7 +31,11 @@ var SEND_GASLIMIT = 250000;
 var SEND_GASPRICE = 0.0000004;
 
 var MAX_BYTES_PER_ARRAY_SLOT = 64;
-var ARRAY_CAPACITY = 10;
+
+var REGEX_BYTES = /bytes([0-9]+)/;
```

Comment:
According to https://solidity.readthedocs.io/en/develop/types.html#reference-types,

Fixed-size byte arrays ranges from `bytes1` to `bytes32` only. So the regex can be stricter

Reply:
done. technically not necessary as the abi is generated from the contract and the contract won't compile if < 1 or > 32. but ya, someone can accidentally manually mess with the ABI types.